wpilibc/src/main/native/include/frc/Solenoid.h

```diff
@@ -61,6 +61,32 @@ class Solenoid : public wpi::Sendable, public wpi::SendableHelper<Solenoid> {
    */
   bool Get() const;
```

Comment:
`Get` here should be deprecated too, I think

Reply:
fixed